app/features-json/setup_json_controller.rb

```diff
@@ -8,5 +8,19 @@ class SetupJSONController < APIController
     get "#{HOME}/configured", authenticate: false do
       return json(Services.onboarding_service.correct_setup?)
     end
+
+    get "#{HOME}/userDetails" do
```

Comment:
I think for the most part we've been using `snake_case` instead of `camelCase` for our routes. Since `snake_case` seems to be the dominant format, I suggest we stay consistent with it.

Reply:
Ah yeah, totally